orion/pkg/tests_integration/semester_integration_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // Test: Create 3 Semesters and GetAll()
 func Test_CreateSemesters(t *testing.T) {
-	resetTable(t, domains.TABLE_SEMESTERS)
+	resetSessionTables(t)
```

Comment:
Again, kind of weird, that we are reseting a bunch of tables when our test is only operating on the Semesters table.
If things are affecting this test when it shouldn't, I think there may be an underlying bug? Before this test, the SEMESTERS table should be cleared.

And before another test, it would be safe to reset used tables for that test.